test/sortByProps.js

```diff
@@ -0,0 +1,57 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('sortByProps', function() {
+  const alice = {
+    name: 'ALICE',
+    age: 101,
+    lastName: 'Smith',
+  };
+  const bob = {
+    name: 'Bob',
+    age: -10,
+    lastName: 'Smith',
+  };
+  const clara = {
+    name: 'clara',
+    age: 314.159,
+    lastName: 'Kris',
+  };
+  const people = [clara, bob, alice];
+
+  context('given an array of objects', function() {
+    specify(
+      'property to sort by exists, should return sorted values',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['name']), [alice, bob, clara]);
+      }
+    );
+
+    specify(
+      'property to sort by does not exist, should return original list',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['p']), people);
+      }
+    );
+
+    specify(
+      'sort by multiple properties, should return sorted values',
+      function() {
+        assert.deepEqual(RA.sortByProps(people)(['lastName', 'name']), [
+          clara,
+          alice,
+          bob,
+        ]);
+      }
+    );
+
+    specify('ignore properties that do not exist', function() {
```

Comment:
```suggestion
    specify('should ignore properties that do not exist', function() {
```

Reply:
The rule in tests is to always start with world `should`. That way tests are consistent and easy to read because you expect the senteces in standard format.